desc/equilibrium/equilibrium.py

```diff
@@ -973,7 +973,6 @@ def need_src(name):
             # Warn if best way to compute accurately is increasing resolution.
```

Comment:
Adjusted logic to be more self-consistent. Will raise all warnings raised previously and covers some additional cases.